dynamic-rapidpro-workspace-settings/utils.js

```diff
@@ -51,41 +48,57 @@ const getInputs = (core) => {
   return inputs;
 };
 
-const getFormattedFlows = flows => `module.exports = ${util.inspect(flows)};\n`;
-
-const isValidFlows = data => Object.keys(data).length !== 0 && data.constructor === Object;
+const isValidObject = data => Object.keys(data).length !== 0 && data.constructor === Object;
 
 const run = async (githubWorkspacePath, params, fs) => {
   try {
     if (!githubWorkspacePath) {
       throw new Error(`GITHUB_WORKSPACE not defined`);
     }
     const secrets = getInputs(params);
-    if (!isValidFlows(secrets.rp_flows)) {
-      throw new Error(`Invalid flows data`);
+    
+    if (!isValidObject(secrets.outbound_mapping_exprs)) {
+      throw new Error(`Invalid outbound mapping expressions data`);
     }
     const codeRepository = path.resolve(path.resolve(githubWorkspacePath), secrets.directory);
     process.chdir(codeRepository);
     const url = getCouchDbUrl(secrets.hostname, secrets.couch_node_name, secrets.value_key, secrets.couch_username, secrets.couch_password);
     const settingsFile = fs.existsSync(baseSettingsFile) ? baseSettingsFile : appSettingsFile;
     const appSettings = fs.readFileSync(`${codeRepository}/${settingsFile}`, 'utf8');
-    const settings = await getReplacedContent(JSON.parse(appSettings), secrets);
+    const settings = await getReplacedContent(JSON.parse(appSettings), flatten(secrets));
+    let flowsContent = null;
+    if (fs.existsSync(flowsFile)) {
+      const rawFlowsContent = fs.readFileSync(`${codeRepository}/${flowsFile}`, 'utf8');
+      flowsContent = await getReplacedContent(rawFlowsContent, flatten(secrets));
+    }
 
-    await axios.put(url.href, `"${secrets.rp_api_token}"`);    
+    await axios.put(url.href, `"${secrets.rp_api_token}"`);
     fs.writeFileSync(`${codeRepository}/${settingsFile}`, settings);
-    fs.writeFileSync(`${codeRepository}/${flowsFile}`, getFormattedFlows(secrets.rp_flows));
+    fs.writeFileSync(`${codeRepository}/${flowsFile}`, flowsContent);
     core.info(`Successful`);
   } catch (error) {
     core.setFailed(error.message);
   }
 };
 
+const flatten = obj => {
```

Comment:
consider giving it a name that makes it easier to understand it's function. e.g flattenObject.

You should probably throw if obj is not an object